### ISSUE TITLE
DogeCoin

### ISSUE THREAD

**Issue by simpatbos** [benawad/dogehouse#1689]
**Describe the feature you'd like**
A currency earned from participating in dogehouse rooms. You can then spend these coins on special features, maybe a gadget that goes next to your name (a badge or title type of thing). This would incentivize people to stay engaged with dogehouse.

**Additional Context**
This idea is implemented very well on twitch. Channel points are often the thing that makes me stay watching a streamer, and the gambling is addictive.

**Comment by kasparnoor** [comment]
Good idea, but we don't want people sitting in rooms for no reason to my knowledge.

**Comment by poweitsao** [comment]
I think instead of rewarding people sitting in rooms to earn dogecoin, maybe allow listeners to tip the mods/speaker in dogecoin for their effort. Potential implementation could look like this: users all get a dogecoin wallet associated with their account and people can tip them/load up the r wallet to tip others.

**Comment by TheOtterlord** [comment]
I believe ben plans to use [dogecoin](https://dogecoin.com/) in monetisation somehow, but monetisation will not be coming until dogehouse is a complete stable platform.
